src/alda/lisp/attributes.clj

```diff
@@ -56,7 +44,6 @@
    have a duration of 450 ms, and the next event will be set to occur in 500 ms."
   :aliases [:quant :quantize]
   :initial-val 0.9
-  :fn-name quant
```

Comment:
Deleting this line, as the changes to `defattribute` make it so that functions are defined for `quantization`, `quant` and `quantize` based on the attribute name and aliases. (Note: If I had left it in, an alias would not be defined for `quantization`)